tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Remote Staking via XCM
+description: In this guide, we'll be leveraging remote execution to remotely stake GLMR on Moonbeam using a series of XCM instructions.
+---
+
+# Remote Staking via XCM
+
+![Banner Image](/images/tutorials/remote-staking-via-xcm/remote-staking-via-xcm-banner.png)
+
+## Introduction {: #introduction } 
+In this tutorial, we’ll stake DEV tokens remotely by sending XCM instructions from an account on the Moonbase Relay Chain (equivalent to the Polkadot Relay Chain). This tutorial assumes a basic familiarity with [XCM](/builders/xcm/overview/){target=_blank} and [Remote Execution via XCM](builders/xcm/xcm-transactor/){target=_blank}. You don’t have to be an expert on these topics but you may find it helpful to have some XCM knowledge as background. 
+
+There are actually two possible approaches for staking on Moonbeam remotely via XCM. We could send a remote EVM call that calls the [staking precompile](builders/pallets-precompiles/precompiles/staking/){target=_blank}, or we could use XCM to call the [parachainStaking pallet](builders/pallets-precompiles/pallets/staking/){target=_blank} directly without interacting with the EVM. For this tutorial, we’ll be taking the latter approach and interacting with the parachainStaking pallet directly. 
```

Comment:
Use the standards we normally apply here:

```
- All UI elements should be in bold. This is done in some places, but other places it looks like it was forgotten
- Typically on the docs site, code-related variables, functions, addresses, and file names are typically in back ticks but for blog posts we italicize these items (unless they are a UI element, then they should be bold)
- Staking Precompile doesn't need to be capitalized
- WEI should be Wei
```

So I guess it should be `parachainStaking`